libcst/_nodes/tests/test_import.py

```diff
@@ -397,9 +397,11 @@ class ImportFromCreateTest(CSTNodeTest):
                         cst.ImportAlias(cst.Name("bar"), comma=cst.Comma()),
                         cst.ImportAlias(cst.Name("baz"), comma=cst.Comma()),
                     ),
+                    lpar=cst.LeftParen(),
+                    rpar=cst.RightParen(),
                 ),
-                "code": "from foo import bar,baz,",
-                "expected_position": CodeRange((1, 0), (1, 23)),
+                "code": "from foo import (bar,baz,)",
```

Comment:
Curious if this is intentional -- the old syntax w/o parens is valid.  Does the new parser reject it?